packages/@azure/servicebus/data-plane/examples/javascript/advancedFeatures/deferral.js

```diff
@@ -0,0 +1,130 @@
+/*
+  This sample demonstrates how the defer() function can be used to defer a message for later processing.
+
+  In this sample, we have an application that gets cooking instructions out of order. It uses
+  message deferral to defer the instruction that is out of order, and then processes it in order.
+
+  See https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-deferral to learn about
+  message deferral.
+*/
+
+const Namespace = require("@azure/service-bus").Namespace;
+const delay = require("@azure/service-bus").delay;
```

Comment:
@bterlson AFAIK there shouldn't be a problem using require twice like this on the same module. Can you confirm? Also, is this the best way to go about this or you have any suggestions for importing multiple objects from the same module other than the below
```
const serviceBus = require("@azure/service-bus");
const Namespace = serviceBus.Namespace;
const delay = serviceBus.delay;
```

Reply:
No problem technically, but it is a little suspect. I prefer:

```
const { Namespace, delay } = require("@azure/service-bus")
```

Reply:
Cool, I thought that way was supported only in the `import` statement :)